src/app/drive/components/DriveExplorer/DriveExplorer.tsx

```diff
@@ -301,9 +302,11 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
     (item: DriveItemData | AdvancedSharedItem) => {
       if (item.isFolder) {
         dispatch(storageThunks.goToFolderThunk({ name: item.name, id: item.id }));
+        navigationService.pushFolder(item.uuid);
```

Comment:
Is this added because it is not navigating to the folder? 
If `goToFolderThunk` is not working better add the change inside the thunk, not outside.

Reply:
I've noticed that we use the `pushFolder` outside the `goToFolderThunk` in some cases, for example in Breadcrumbs. I'm not sure if its better to add `pushFolder` inside of `goToFolderThunk`

Reply:
Mmm, we should revise it to standardise in one place the way to navigate through the folders.